### PR TITLE
Package Testing job for PS 8.1

### PR DESCRIPTION
**Pull request by kaushikpuneet07**

### CODE REVIEW

File: ps/jenkins/package-testing-ps-build-8.1.groovy [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    '''
+}
+
+setup_amazon_package_tests = { ->
+    sh '''
+        sudo amazon-linux-extras install epel
+        sudo yum -y update
+        sudo yum install -y ansible git wget
+    '''
+}
```
Comment (EvgeniyPatlan) [inline ps/jenkins/package-testing-ps-build-8.1.groovy:28]: it is the same as `setup_rhel_package_tests`

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

**Comment by kaushikpuneet07** [comment]
Fixed review comments from evgeniy

File: ps/jenkins/package-testing-ps-8.1.groovy [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+List nodes_to_test = []
+if (params.node_to_test == "all") {
+    nodes_to_test = all_nodes
+} else {
+    nodes_to_test = [params.node_to_test]
+}
+
+void runNodeBuild(String node_to_test) {
+    build(
+        job: 'package-testing-ps81-build',
```
Comment (eleo007) [inline ps/jenkins/package-testing-ps-8.1.groovy:29]: Are we going to create separate jobs for every innovations releases?

File: ps/jenkins/package-testing-ps-8.1.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            num-to-keep: 10
+    pipeline-scm:
+        scm:
+            - git:
+                url: https://github.com/Percona-Lab/jenkins-pipelines.git
+                branches:
+                    - 'master'
+                wipe-workspace: true
+        lightweight-checkout: true
+        script-path: ps/jenkins/package-testing-ps-8.1.groovy
```
Comment (eleo007) [inline ps/jenkins/package-testing-ps-8.1.yml:20]: The same as above, are we going to have new jobs on every innovations release?

File: ps/jenkins/package-testing-ps-build-8.1.groovy [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+    "install",
+    "upgrade",
+    "maj-upgrade-to",
+    "kmip",
+    "kms",
+]
+
+product_action_playbooks = [
+    ps81: [
+        install: "ps_81.yml",
```
Comment (eleo007) [inline ps/jenkins/package-testing-ps-build-8.1.groovy:67]: The same here: are we going to create/update playbooks on every innovations release?

File: ps/jenkins/package-testing-ps-build-8.1.groovy [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+    agent none
+
+    options {
+        skipDefaultCheckout()
+    }
+
+    parameters {
+        choice(
+            name: "product_to_test",
+            choices: ["ps81", "client_test"],
```
Comment (eleo007) [inline ps/jenkins/package-testing-ps-build-8.1.groovy:123]: The same as above

File: ps/jenkins/package-testing-ps-8.1.yml [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    parameters:
+        - choice:
+            name: product_to_test
+            choices:
+                - "ps81"
```
Comment (eleo007) [inline ps/jenkins/package-testing-ps-8.1.yml:25]: The same as above

File: ps/jenkins/package-testing-ps-8.1.groovy [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+    )
+}
+
+pipeline {
+    agent none
+
+    parameters {
+        choice(
+            name: "product_to_test",
+            choices: ["ps81", "client_test"],
```
Comment (eleo007) [inline ps/jenkins/package-testing-ps-8.1.groovy:49]: The same as above

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**